databricks_cli/instance_pools/cli.py

```diff
@@ -0,0 +1,157 @@
+              help='File containing JSON request to POST to /api/2.0/instance_pools/create.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/instance_pools/create'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_cli(api_client, json_file, json):
+    """
+    Creates a Databricks instance pool.
+
+    The specification for the request json can be found at
+    https://docs.databricks.com/api/latest/instance-pools.html#create
+    """
+    json_cli_base(json_file, json, lambda json: InstancePoolsApi(api_client).create_instance_pool(json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST to /api/2.0/instance_pools/edit.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/instance_pools/edit'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def edit_cli(api_client, json_file, json):
+    """
+    Edits a Databricks instance pool.
+
+    The specification for the request json can be found at
+    https://docs.databricks.com/api/latest/instance-pools.html#edit
+    """
+    if not bool(json_file) ^ bool(json):
+        raise RuntimeError('Either --json-file or --json should be provided')
+    if json_file:
+        with open(json_file, 'r') as f:
+            json = f.read()
+    deser_json = json_loads(json)
+    InstancePoolsApi(api_client).edit_instance_pool(deser_json)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--instance-pool-id', required=True, type=InstancePoolIdClickType(),
+              help=InstancePoolIdClickType.help)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_cli(api_client, instance_pool_id):
+    """
+    Removes a Databricks instance pool given its ID.
+
+    The instance pool is removed asynchronously. Once the deletion has completed,
+
+    Use ``databricks instance_pools get --instance-pool-id INSTANCE_POOL_ID`` to check
+    termination states.
+    """
+    InstancePoolsApi(api_client).delete_instance_pool(instance_pool_id)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--instance-pool-id', required=True, type=InstancePoolIdClickType(),
+              help=InstancePoolIdClickType.help)
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_cli(api_client, instance_pool_id):
+    """
+    Retrieves metadata about a instance pool.
+    """
+    click.echo(pretty_format(InstancePoolsApi(api_client).get_instance_pool(instance_pool_id)))
+
+
+def _instance_pools_to_table(instance_pools_json):
+    ret = []
+    for c in instance_pools_json.get('instance_pools', []):
+        ret.append((c['instance_pool_id'], truncate_string(c['instance_pool_name']), c['stats']))
```

Comment:
I noticed this might look like

```
0628-224956-spout32-pool-w4YRAcnX   monkey                       {u'idle_count': 0, u'pending_used_count': 0, u'pending_idle_count': 0, u'used_count': 0}
```
Can we remove the `u` from the JSON?